packages/backend/src/api/resources/Domains.ts

```diff
@@ -0,0 +1,9 @@
+import type { DomainJSON } from './JSON';
+
+export class Domains {
+  constructor(readonly id: string, readonly name: string) {}
+
```

Comment:
❓ I'd expect a delete method here, am I missing something ?

Reply:
@panteliselef the convention here is to have the methods in the `/endpoints` folder